pages/team.js

```diff
@@ -0,0 +1,25 @@
+import Head from "next/head"
+import Layout from "components/layout"
+import { Image, Heading  } from "@chakra-ui/react"
+import Member from "components/teamMember"
+import team from "../team"
+
+export default function Team() {
+  const members = Object.values(team)
+  return (
+    <Layout>
+      <Head>
+        <meta charSet="UTF-8" />
+        <meta httpEquiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Common Prefix - Team</title>
+        <link rel="icon" type="image/png" href="/philosopher-stone.png" />
+      </Head>
```

Comment:
DRY, let's create a separate wrapper that includes this metadata for all pages (flare, harmony, homepage, team, etc.)

Reply:
Agreed - I was planning on creating a separate PR for such DRY/cleanup changes